release.nix

```diff
@@ -2,8 +2,9 @@
 
 
 with nixpkgs.pkgs; rec {
-  workshop1 = pkgs.haskellPackages.callPackage ./workshop1 {};
-  workshop1-slides = pkgs.callPackage ./workshop1/slides {};
+  workshop1-slides = import ./workshop1/slides {};
+  workshop1-cheatsheet = import ./workshop1/cheatsheet {};
```

Comment:
Could you do `pkgs.callPackage ./workshop1/slides {}` instead? The nixpkgs would then be shared from the toplevel